### DISCUSSION TITLE
{gtfsio}: read/write GTFS transit feeds in R

### DISCUSSION THREAD

**Discussion by dhersz** [r-transit/gtfsio#2] [General]
Hello @tbuckl, @mpadge, @polettif and @rafapereirabr.

Following up our discussion, I have started the `{gtfsio}` package repo. It only has a single function as of now, `get_gtfs_standards()`, to generate the R standards on how to best represent GTFS files and fields. I tried to stick to the format @mpadge introduced in [`gtfsrouter::get-reference-fields()`](https://github.com/ATFutures/gtfs-router/blob/master/R/gtfs-reference-fields.R), but I also added a third field in each field-specifying `list` to describe the values it can assume (mainly relevant to `ENUM`s). I'd like to hear your thoughts on this.

I plan to put in some work on the package in the following weeks, but obviously feel free to play with it yourselves as well.

Cheers!

**Comment by mpadge** [comment]
Awesome work @dhersz, thank!!! I'll have a look, and definitely be chipping in to help whenever I can

**Reply by tbuckl** [reply]
agreed, this is awesome @dhersz. ideally we would just replace IO in tidytransit and import gtfsio. i love it!

**Comment by dhersz** [comment]
I gave you all access to write to the repo. I thought that you'd have it by default by being an `r-transit` member, but apparently not.
